src/main/scala/com/imaginea/activegrid/core/models/Neo4jRep.scala

```diff
@@ -0,0 +1,21 @@
+package com.imaginea.activegrid.core.models
+
+import org.neo4j.graphdb.Node
+
+/**
+ * Created by babjik on 4/10/16.
+ */
+trait Neo4jRep[T] {
+  def toNeo4jGraph(entity: T): Option[Node]
+
+  def fromNeo4jGraph(nodeId: Long): T
+}
+
+trait Neo4jRep2[T] {
```

Comment:
People are using `def fromNeo4jGraph(nodeId: Long): T` returning T as return type
where as Neo4jRep2 returning Option[T]
`def fromNeo4jGraph(nodeId: Long): Option[T]`

If you add your comments on this return type I can merge both trait